src/server/saved_objects/client/lib/parse_es_doc.js

```diff
@@ -0,0 +1,16 @@
+import { get } from 'lodash';
+
+export function parseEsDoc(doc, overrides = {}) {
```

Comment:
Hm, is "normalize" better than "parse"? Might not be, just a thought I had while scanning through the code